### PR TITLE
Deploy prism & xprism

### PR DESCRIPTION
**Pull request by steven4354**

### CODE REVIEW

File: src/integration_tests/deploy_prism/config.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    token_code_id: number,
+}
+
+// source: 
+// https://finder.terra.money/mainnet/address/terra1h4al753uvwmhxwhn2dlvm9gfk0jkf52xqasmq2
+export function PrismGovConfig(
+    prism_token_addr: Addr,
+    token_code_id: number,
+): PrismGovConfig {
+    // TODO: add prod vs test
```
Comment (pronvis) [inline src/integration_tests/deploy_prism/config.ts:44]: please, fix this TODO :)

File: src/utils.ts [hunk 2]
```diff
@@ -299,9 +313,17 @@ export async function calc_fee_and_send_tx(lcd_client: LCDClient, sender: Wallet
 
 async function get_tx_fee(lcd_client: LCDClient, sender: Wallet, msgs: Msg[], tax?: Coin[]): Promise<StdFee | undefined> {
 	try {
+		let gasAdjustment;
+
+		if (is_localterra(lcd_client)) {
```
Comment (pronvis) [inline src/utils.ts:318]: I think it should be
```
		if (is_localterra(lcd_client)) {
			gasAdjustment = 2.0;
		} else {
			gasAdjustment = 1.2;
		}
```

**Comment by pronvis** [comment]
In general, as a raw version, I think it is fine

File: src/utils.ts (hunk 2 above)
Comment (steven4354) [reply src/utils.ts:318]: sg updated 👍

**Pull request merged**